.env

```diff
@@ -0,0 +1 @@
+GA_TRACKING_ID=UA-96772907-1
```

Comment:
This file should be ignored, how come is commited? I can see it on my local .gitignore...